docker-compose.yml

```diff
@@ -211,3 +211,7 @@ services:
     image: dyne/w3c-did-driver:latest
     ports:
       - "8133:8080"
+  did-jwk-driver:
+    image: transmute/restricted-resolver:latest
+    ports:
+      - "8134:8080"
```

Comment:
I don't understand this part, exactly... normally I run this container like this:

```
docker pull transmute/restricted-resolver
docker run -d -p 8080:8080 transmute/restricted-resolver
```

Reply:
Hmm I think the idea of incrementing port numbers with each driver in docker-compose was that you can then reach each driver separately on the host machine.

In other words, the did:sov, did:v1, did:jwk, did:web etc. Docker containers can all be reached "directly" via different ports on the same host, without have to go through the Universal Resolver "main" container.

Reply:
ok, so I think this should be good to go.